sdk/identity/azcertificatemanager/go.mod

```diff
@@ -0,0 +1,21 @@
+module github.com/Azure/azure-sdk-for-go/sdk/identity/azcertificatemanager
+
+go 1.18
+
+require (
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.0.0
```

Comment:
Please update this (and other dependencies) to the latest version.

`go get -u`

Please then execute `go mod tidy` and included the generated `go.sum` file.